openedx/core/djangoapps/credit/tasks.py

```diff
@@ -133,21 +146,18 @@ def _is_credit_requirement(xblock):
         True if XBlock is a credit requirement else False
 
     """
-    is_credit_requirement = False
-
-    if callable(getattr(xblock, "is_course_credit_requirement", None)):
-        is_credit_requirement = xblock.is_course_credit_requirement()
-        if is_credit_requirement:
-            if not callable(getattr(xblock, "get_credit_requirement_namespace", None)):
-                is_credit_requirement = False
-                LOGGER.error(
-                    "XBlock %s is marked as a credit requirement but does not "
-                    "implement get_credit_requirement_namespace()", unicode(xblock)
-                )
-            if not callable(getattr(xblock, "get_credit_requirement_name", None)):
-                is_credit_requirement = False
-                LOGGER.error(
-                    "XBlock %s is marked as a credit requirement but does not "
-                    "implement get_credit_requirement_name()", unicode(xblock)
-                )
-    return is_credit_requirement
+    if not callable(getattr(xblock, "get_credit_requirement_namespace", None)):
+        LOGGER.error(
```

Comment:
This would get picked up pretty quickly by our existing monitoring for error log messages.

That said, it's really, really unlikely to happen in production.  Even if it did, the user-facing impact is pretty minimal; the requirement just won't be added to the list.  These error messages are mainly to help diagnose problems in local development if/when we add new requirements like this.